src/geop/processing/src/main/scala/com/azavea/mmw/TR55.scala

```diff
@@ -0,0 +1,395 @@
+package com.azavea.mmw
+
+import geotrellis.vector._
+import geotrellis.vector.io.json._
+import geotrellis.raster._
+import geotrellis.raster.histogram._
+import geotrellis.raster.rasterize.{Rasterizer, Callback}
+import geotrellis.spark._
+
+
+import org.apache.spark.SparkContext._
+import scala.collection.mutable
+import spire.syntax.cfor._
+
+object TR55 {
+
+  /** maps a 32-bit word to a soil type */
+  val wordToSoil = collection.immutable.HashMap(
```

Comment:
This is a painfully long enough val that I would suggest pulling it out of this object. Perhaps have a `Constants` object, in it's own file, with an object that has this?